conf/application.conf

```diff
@@ -19,25 +19,9 @@ include "backend.conf"
 
 appName = register-country-by-country-reporting
 
-# An ApplicationLoader that uses Guice to bootstrap the application.
-play.application.loader = "uk.gov.hmrc.play.bootstrap.ApplicationLoader"
-
-# Primary entry point for all HTTP requests on Play applications
-play.http.requestHandler = "uk.gov.hmrc.play.bootstrap.http.RequestHandler"
-
-# Provides an implementation of AuditConnector. Use `uk.gov.hmrc.play.audit.AuditModule` or create your own.
-# An audit connector must be provided.
-play.modules.enabled += "uk.gov.hmrc.play.audit.AuditModule"
-
-# Provides an implementation of MetricsFilter. Use `uk.gov.hmrc.play.graphite.GraphiteMetricsModule` or create your own.
-# A metric filter must be provided
-play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
-
-# Provides an implementation and configures all filters required by a Platform frontend microservice.
-play.modules.enabled += "uk.gov.hmrc.play.bootstrap.backend.BackendModule"
-
 # Default http client
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
```

Comment:
shouldn't we try to remove `HttpClientModule` as part of this ticket? It may require some further updates

Reply:
do you mean to update to HttpClientV2Module? 
I had thought we should do that to the CBC services too after i updated the FI-management service but thought i might come under the 'routine dep updates' ticket we have

Reply:
yes common updates is generally for dependencies updates. For this I believe there may only places you have to update `HttpClient` and could be an easy win.

Reply:
Aye i agree, i'll added em in 👍